src/lib/store.js

```diff
@@ -62,6 +62,7 @@ export default class Store {
 		this.allowedVendorIds = new Set(allowedVendorIds);
 		this.isConsentToolShowing = false;
 		this.isBannerShowing = false;
+		this.theme = config.theme;
```

Comment:
I don't think we need the `theme` in the Store.  It can be fed into the top level components from the config.